firmware/lib/TestStandCommController/src/TestStandCommController.h

```diff
@@ -18,7 +18,9 @@ class TestStandCommController : public TestStandComm
         SerialResult ping();
         SerialResult log(LogLevel log_level, const char *fmt, ...);
         SerialResult status(Status status);
-        SerialResult data(uint8_t *data, uint8_t length);
+        SerialResult position(int32_t x, int32_t y);
```

Comment:
Similar comment, position (assuming in counts) would always be a positive number? Since everything is defined wrt the origin (0,0)?

Reply:
See above